pkg/config/event_listener.go

```diff
@@ -64,6 +66,9 @@ func (e *eventListener) open() {
 	if e.configFilesChangedObserver == nil {
 		e.configFilesChangedObserver = new(observer.Observer)
 		e.configFilesChangedObserver.Open()
+
+		// Buffer file change events, 1e6 ns == 1 ms
+		e.configFilesChangedObserver.SetBufferDuration(1000000 * time.Nanosecond)
```

Comment:
Any reason to not use `1 * time.Millisecond`?

Reply:
> Any reason to not use 1 * time.Millisecond?

@jhernand , No, playing with timings ... changed to `Millisecond` :+1: 

cc @nimrodshn ( who also asked the same question :-) )

Reply:
OK. It looks good to me as is. Should I merge?

Reply:
I'm ok with it :-)